x-pack/solutions/security/plugins/elastic_assistant/server/ai_assistant_data_clients/find.ts

```diff
@@ -41,6 +41,8 @@ export interface FindResponse<T> {
   total: number;
 }
 
+const camelToSnake = (camelStr: string): string =>
+  camelStr.replace(/(?<!^)([A-Z])/g, '_$1').toLowerCase();
```

Comment:
nit: This also exists over in `x-pack/platform/packages/shared/kbn-elastic-assistant/impl/assistant/conversations/conversation_settings_management/index.tsx`, so maybe pull out to common utils? I believe `_.snakeCase()` should provide similar functionality if you'd like.

Reply:
Thanks! https://github.com/elastic/kibana/pull/211831/commits/d7e6ad642a372e1285a9b91660c06a555797fada